pkg/ingester/ingester.go

```diff
@@ -2881,6 +2881,64 @@ func (i *Ingester) flushHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+const (
+	mode     = "mode"
```

Comment:
I think we can remove this since it's only used in the ModeHander func.

Reply:
Removed constants